test/net/sourceforge/kolmafia/KoLCharacterTest.java

```diff
@@ -548,4 +548,16 @@ public void oldQuestDoesNotGiveExperience() {
       }
     }
   }
+
+  @Nested
+  class Liberation {
+    @Test
+    void resizesOrganContentsAfterSmallPath() {
+      try (var cleanups = new Cleanups(withPath(Path.SMALL), withInebriety(1), withFullness(1))) {
```

Comment:
Small can have up to 2 fullness used, which updates to 20 after liberation. Might be interesting to test this as it is above the typical fullness limit

Reply:
Sure. I am just multiplying the number by 10 ;)